daemon/daemon.go

```diff
@@ -30,7 +30,6 @@ import (
 	"github.com/docker/docker/daemon/exec"
 	"github.com/docker/docker/daemon/execdriver"
 	"github.com/docker/docker/daemon/execdriver/execdrivers"
-	"github.com/docker/docker/daemon/graphdriver"
 	_ "github.com/docker/docker/daemon/graphdriver/vfs" // register vfs
```

Comment:
OK. The problem is that reference store has a dependency on image because of the `image.ID` type, right? We should either move the layer/image/ref types to a common package or reference store should just work with plain digests.